### PR TITLE
change filter to ModelAutocomplete

### PR DESCRIPTION
**Pull request by Raulnet**
change filter to ModelAutocomplete

### CODE REVIEW

File: src/Admin/JecouteSurveyAdmin.php [hunk 1]
```diff
@@ -49,13 +50,21 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
...
                 'show_filter' => true,
             ])
-            ->add('author.referentTags', null, [
+            ->add('author.referentTags', ModelAutocompleteFilter::class, [
                 'label' => 'Zones',
                 'show_filter' => true,
+                'field_options' => [
+                    'model_manager' => $this->getModelManager(),
+                    'admin_code' => $this->getCode(),
+                    'property' => 'name',
```
Comment (Remg) [inline src/Admin/JecouteSurveyAdmin.php]: What about adding `'minimum_input_length' => 2,` to be able to search by department number ? 
**before:**
screenshot from 2019-01-31 12-48-20

**after:**
screenshot from 2019-01-31 12-49-12

File: src/Admin/JecouteSurveyAdmin.php [hunk 2]
```diff
@@ -49,13 +50,21 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
...
+                ],
                 'show_filter' => true,
             ])
-            ->add('author.referentTags', null, [
+            ->add('author.referentTags', ModelAutocompleteFilter::class, [
                 'label' => 'Zones',
                 'show_filter' => true,
+                'field_options' => [
+                    'model_manager' => $this->getModelManager(),
+                    'admin_code' => $this->getCode(),
```
Comment (Remg) [inline src/Admin/JecouteSurveyAdmin.php]: why are these 2 lines needed ?

File: src/Admin/JecouteSurveyAdmin.php (hunk 1 above)
Comment (Raulnet) [reply src/Admin/JecouteSurveyAdmin.php]: fixed

File: src/Admin/JecouteSurveyAdmin.php (hunk 2 above)
Comment (Raulnet) [reply src/Admin/JecouteSurveyAdmin.php]: removed

**Pull request merged**